pkg/ddevapp/webserver_config_packr_assets/nginx-site-typo3.conf

```diff
@@ -31,6 +31,12 @@ server {
         try_files $uri $uri/ /index.php$is_args$args;
     }
 
+    # TYPO3 11 Backend URL rewriting support
+    location /typo3/ {
+        absolute_redirect off;
+        try_files $uri $uri/ /typo3/index.php$is_args$args;
```

Comment:
I would recommend to use:
```
try_files $uri /typo3/index.php$is_args$args;
```
The TYPO3 backend does not nested index.php files to be evaluated and directory indexes aren't desired either. Avoid `$uri/` saves a roundtrip for "good" routes.


Also an additional explicit route for `/typo3` would be nice:
```
location = /typo3 {
        return 302 /typo3/;
}
```

Reply:
Thanks @bnf !